app/Http/Controllers/BuslineController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Busline;
+use Illuminate\Http\Request;
+
+class BuslineController extends Controller
+{
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
```

Comment:
You're using PHP 7.4, so you can type almost everything you want:
```
public function create(): Response
```

With this approach, we can skip adding docblocks everywhere.